spec.html

```diff
@@ -41830,6 +41844,22 @@ <h1>
             1. Return ~unused~.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-isfixedlengthtypedarray" type="abstract operation">
```

Comment:
I would put this up with [IsTypedArrayOutOfBounds](https://tc39.es/ecma262/multipage/ordinary-and-exotic-objects-behaviours.html#sec-istypedarrayoutofbounds]). This section is only really for the initialization-related AOs.

Also since this takes a TA, it should be `IsTypedArrayFixedLength`: it asks a question about a TA, rather than determining whether an arbitrary object is a fixed-length TA.

Reply:
Sure, moving it sounds good.

Re: IsTypedArrayFixedLength name, this was to be consistent with IsFixedLengthArrayBuffer. We can change both though as a followup, I have no issue with your suggestion.